docs/source/upcoming_release_notes/1083-DUT_MotionPneumaticActuator.rst

```diff
@@ -0,0 +1,31 @@
+1083 DUT_MotionPneumaticActuator
+#################
+
+API Changes
+-----------
+- Add module pneumatic.py
+- Add test test_pneumatic.py
```

Comment:
Nitpick: this section is really for API breaks, like removing modules. Maybe I should update the template.
```suggestion
- N/A
```